docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -111,13 +110,17 @@ app:
 
 == Profiles
 
-As you can see in the previous snippet, you can use xref:config-reference.adoc#profiles[profiles] in YAML. The profile
-key requires double quotes: `"%test"`. This is because YAML does not support keys starting with `%`.
+As you can see in the previous snippet, you can use link:https://quarkus.io/guides/config-reference#profiles[profiles] in YAML.
```

Comment:
Should use a xref and not the direct link.

Reply:
Updated. Thanks.